### ISSUE TITLE
Image rotation and crop

### ISSUE THREAD

**Issue by izumitelj** [NaitLee/Cat-Printer#34]
Hello! Thanks for this great and simple alternative :). I've noticed one issue while inserting images – source photo orientation is not respected and images in portrait orientation end up rotated -90° (to landscape, thus very small in print). 

It would be great if the app could have basic editing feature like +90° rotation and crop. With those tools we could skip the need for separate image editing apps altogether.

**Comment by sync1211** [comment]
I feel like, if we're adding an image editor we should use that instead of the current import system.
As this editor would need a lot of functionality, we could use an open-source image editor (like [miniPaint](https://github.com/viliusle/miniPaint)).

**Comment by izumitelj** [comment]
@sync1211 the miniPaint looks really nice. 

I didn't have that in my mind particularly. I see this project as minimalist straight-forward replacement for the cluttered official app. I only suggested these essential features because I would make possible to reasonably control downloaded and original images/photos – which I believe is how most users use this app and printer. I see it more as extension of printer settings (portrait/landscape), with no need to ever save the image file back.

**Comment by NaitLee** [comment]
That's right, I prefer keeping it minimal, and our work is just to: interact with our lovely printer, neatly.

miniPaint is clean! And the best is to just use it via its GitHub Pages instance.
In the mean while I found [Pocket Paint](https://github.com/Catrobat/Paintroid), an Android app. It's also minimal.
For Desktop, stuffs like GIMP and Krita are there.
Similarly, in CLI, the project makes use of ImageMagick and Ghostscript, rather than including Pillow/NumPy/...

In contrast, think about why those cluttered apps don't satisfy people. They are simply including & doing too much.
There are good apps around, focusing their supposed job. Instead of "libraries", just put apps alongside and enjoy, simple like that! 😉

We should do one thing and do it well. And may check [contributing guideline](https://github.com/NaitLee/Cat-Printer/blob/main/CONTRIBUTING.md).
I think I should state these briefly in docs, too.

Despite saying, text on canvas or rotating/cropping image are probably essential basics. Let's just do it 👌

ps. I want to try if I can do these without scrambling the UI too much, that means to cost time for investigation.

**Comment by NaitLee** [comment]
The newest 0.6 release contains Image Rotation.

I want to make that canvas can control more.
This may make Text Inputting better, and Cropping may come next with it.
